File: ts-sdk/src/sdk/models/shared/walletsvolume.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class WalletsVolume extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=balance" })
  balance: number;

  @SpeakeasyMetadata({ data: "json, name=input" })
  input: number;

  @SpeakeasyMetadata({ data: "json, name=output" })
  output: number;
}

File: python-sdk/src/formanceapi/models/shared/walletsvolume.py
import dataclasses
from dataclasses_json import dataclass_json
from formanceapi import utils


@dataclass_json
@dataclasses.dataclass
class WalletsVolume:
    balance: int = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('balance') }})
    input: int = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('input') }})
    output: int = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('output') }})
    